httpclient/client.go

```diff
@@ -0,0 +1,42 @@
+package httpclient
```

Comment:
The current project's name is `http-handlers-go`, which implies that it is a backend library. A package such as `httpclient` should be placed in a client library.